lib/pages/class_feedback/service/remote_config.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+
+const String _feedbackEnabled = 'feedback_enabled';
+
+class RemoteConfigService {
+  RemoteConfigService({RemoteConfig remoteConfig})
+      : _remoteConfig = remoteConfig;
+  final RemoteConfig _remoteConfig;
+  final defaults = <String, dynamic>{_feedbackEnabled: false};
+  static RemoteConfigService _instance;
+
+  static Future<RemoteConfigService> getInstance() async {
+    return _instance ??= RemoteConfigService(
+      remoteConfig: await RemoteConfig.instance,
+    );
+  }
+
+  bool get feedbackEnabled =>
+      // ignore: avoid_bool_literals_in_conditional_expressions
+      _remoteConfig == null ? true : _remoteConfig.getBool(_feedbackEnabled);
+
+  Future<dynamic> initialise() async {
+    try {
+      await _remoteConfig.setDefaults(defaults);
+      await _fetchAndActivate();
+    } on FetchThrottledException catch (e) {
+      print('Remote config fetch throttled: $e');
+    } catch (e) {
+      print(
+          'Unable to fetch remote config. Cached or default values will be used.');
+    }
+  }
+
+  Future<dynamic> _fetchAndActivate() async {
```

Comment:
I don't really see the point of this. It's only used once and it's just two function calls, please do the function calls directly in `initialise`.